### PR TITLE
Rw/score bug fix

### PR DESCRIPTION
**Pull request by joeconwaystk**

### CODE REVIEW

File: Chirp/STKScoreboardViewController.m [hunk 1]
```diff
@@ -181,12 +181,43 @@ - (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView cell
...
+            }
+            
             [[c label] setHidden:NO];
         } else if([collectionView numberOfItemsInSection:0] - [indexPath row] - 1 < [[[self game] players] count]) {
             [[c label] setHidden:NO];
             NSInteger playerIndex = [indexPath row] % ([[[self game] players] count] + 1) - 1;
             [[c label] setText:[NSString stringWithFormat:@"%ld", [[[[self game] players] objectAtIndex:playerIndex] score]]];
-        
+            
+            if ([[[[self game]players]objectAtIndex:playerIndex ]score] == 0) {
```
Comment (joeconwaystk) [inline Chirp/STKScoreboardViewController.m]: Again here, this logic says "every time you refresh the cells for any reason, check to see if this players score is greater than zero and then flash it red". It'll always flash red anytime the collection view is updated (which happens often, and not just for score changes) and it'll also always flash when the score is > 0, which means once points have been scored, it'll continue to flash

**Comment by rosshambrick** [comment]
What's the state of this PR. Is it something we should try to get merged?

**Comment by joeconwaystk** [comment]
I think this is discardable...

**Comment by rosshambrick** [comment]
Cool.  I'll go ahead and close it to clear it off the queue.

**Pull request closed without merging**